scripts/crowdin/upload-screenshots.js

```
@@ -0,0 +1,272 @@
+
+      try {
+        const loginFrameLocator = page.locator('#jipt-login-panel iframe');
+        if (await loginFrameLocator.count() > 0) {
+          const loginFrame = await loginFrameLocator.contentFrame();
+          await loginFrame.getByRole('button', { name: 'Log In' }).click({ timeout: 5000 });
+
+          await page.getByLabel('Email or Username').fill(CROWDIN_USERNAME);
+          await page.getByLabel('Password').fill(CROWDIN_PASSWORD);
+          await page.getByRole('button', { name: 'Log In' }).click();
+
+          try {
```

ESLint

[error] Expected indentation of 20 spaces but found 10.

(indent)